lib/api-client.js

```diff
@@ -32,7 +32,7 @@ SlackAPIClient.prototype.SlackError = SlackError;
  * @param {Function} [callback] An optional callback to propagate the response to.
  * @callback {[Error, ResponseBody, Response]}
  */
-SlackAPIClient.prototype.api = function(method, options, callback) {
```

Comment:
Agreed, do you like `.call()` more than `.send()`? Whatever it is, I want it to be the same for both API & Webhook clients